package/MDAnalysis/topology/ITPParser.py

```diff
@@ -27,7 +27,7 @@
 
 Reads a GROMACS ITP_ or TOP_ file to build the system. The topology will
 contain atom IDs, segids, residue IDs, residue names, atom names, atom types,
-charges, chargegroups, masses (guessed if not found), moltypes, and molnums. 
+charges, chargegroups, masses (if found) moltypes, and molnums.
```

Comment:
```suggestion
charges, chargegroups, masses, moltypes, and molnums.
Any masses that are in the file will be read; any missing values will be guessed.
```

Just a suggestion to make it a bit clearer -- it's not obvious to the user how the logic works, and without this note I'm worried they might assume if the masses aren't in the file they aren't in the topology, or that all the masses are guessed if any of them are missing.